subcommand/acl-controller/command.go

```diff
@@ -479,3 +561,71 @@ func (c *Command) writeOptions() *api.WriteOptions {
 	}
 	return nil
 }
+
+// consulDatacenterList returns the current datacenter name and the primary datacenter using the
+// /agent/self API endpoint.
+func (c *Command) consulDatacenterList(agentConfig AgentConfig) (string, string, error) {
+	if agentConfig.Config.Datacenter == "" {
+		return "", "", fmt.Errorf("agent config does not contain Config.Datacenter key: %+v", agentConfig)
+	}
+	if agentConfig.Config.PrimaryDatacenter == "" && agentConfig.DebugConfig.PrimaryDatacenter == "" {
+		return "", "", fmt.Errorf("both Config.PrimaryDatacenter and DebugConfig.PrimaryDatacenter are empty: %+v", agentConfig)
+	}
+	if agentConfig.Config.PrimaryDatacenter != "" {
+		return agentConfig.Config.Datacenter, agentConfig.Config.PrimaryDatacenter, nil
+	} else {
+		return agentConfig.Config.Datacenter, agentConfig.DebugConfig.PrimaryDatacenter, nil
+	}
+}
+
+type AgentConfig struct {
+	Config      Config
+	DebugConfig Config
+}
+
+type Config struct {
+	Datacenter                      string `mapstructure:"Datacenter"`
+	PrimaryDatacenter               string `mapstructure:"PrimaryDatacenter"`
+	MeshGatewayWANFederationEnabled bool   `mapstructure:"ConnectMeshGatewayWANFederationEnabled"`
+}
+
+type templateData struct {
+	Enterprise bool
+}
+
+func (c *Command) templateData() templateData {
+	return templateData{Enterprise: c.flagPartitionsEnabled}
+}
+
+func (c *Command) anonymousPolicyRules() (string, error) {
+	rules := `
+{{- if .Enterprise }}
+namespace_prefix "" {
```

Comment:
Should this have the `partition_prefix ""` scope ([consul-k8s for reference](https://github.com/hashicorp/consul-k8s/blob/aa05d3db2158e0887f49a0f5354893603ff6b6f2/control-plane/subcommand/server-acl-init/rules.go#L118-L137))?